source/fundamentals/data-formats/guid-serialization.txt

```diff
@@ -0,0 +1,173 @@
+- If you call the ``BsonBinaryData.ToGuid()`` method without the ``GuidRepresentation``
+  parameter, it will work only on GUIDs of subtype 4.
+
+.. note::
+   
+   When version 3 of the {+driver-short+} is released, support for ``GuidRepresentationMode.V2``
+   will be removed from the driver and ``V3`` will become the default. 
+
+.. note::
+
+   You can't use both ``GuidRepresentationMode.V2`` and ``GuidRepresentationMode.V3`` 
+   in a single application.
+
+Serializing GUIDs in V3
+-----------------------
+
+``GuidRepresentationMode.V3`` handles GUID serialization at the level of individual
+properties. This mode is more flexible than ``V2``, but it also means you must ensure that 
+each GUID field is serialized and deserialized correctly.
+
+If you're using the {+driver-short+} to :ref:`automap your {+language+} classes to document schemas <csharp-automapping>`,
+you can use the ``BsonGuidRepresentation`` attribute on a GUID property 
+to specify the representation:
+
+.. code-block:: csharp
+
+   public class Widget
+   {
+       public int Id { get; set; }
+
+      [BsonGuidRepresentation(GuidRepresentation.Standard)]
+      public Guid G { get; set; }
+}
+
+.. note::
+
+   ``GuidRepresentation.Standard`` is equivalent to ``BsonBinaryData`` subtype 4.
+
+If you're writing your own serialization and deserialization code, you can use the 
+``GuidSerializer`` class to ensure that the driver handles GUIDs correctly.
+When you construct a ``GuidSerializer``, use the ``GuidRepresentation`` parameter
+to specify the GUID representation. 
+
+The following code sample creates an instance of ``GuidSerializer`` 
+for serializing GUID representations of subtype 4: 
+
+.. code-block::
+
+   var guidSerializer = new GuidSerializer(GuidRepresentation.Standard);
+
+If most of your GUIDs use the same representation, you can register a global 
+``GuidSerializer`` for this representation, then use the ``BsonGuidRepresentation``
+attribute for the other GUIDs. To create and register a ``GuidSerializer``, run the 
+following code early in your application:
+
+.. code-block:: csharp
+
+   BsonSerializer.RegisterSerializer(new GuidSerializer(GuidRepresentation.Standard));
+
+Serializing Objects in V3
+-------------------------
+
+To ensure that GUIDs in objects are serialized and deserialized correctly when using 
+``V3``, you should select the correct GUID representation when constructing your 
+``ObjectSerializer``. The following code sample shows how to 
+create an ``ObjectSerializer`` for a GUID representation of subtype 4:
+
+.. code-block:: csharp
+
+   var objectDiscriminatorConvention = BsonSerializer.LookupDiscriminatorConvention(typeof(object));
+   var objectSerializer = new ObjectSerializer(objectDiscriminatorConvention, GuidRepresentation.Standard);
+
+If your application relies on an ``ObjectSerializer`` to serialize any GUIDs, you 
+must also register the serializer early in your application. The serializer that you 
```

Comment:
If you are writing a custom serializer, which you would do any time you want custom behavior, you have to register it. Is that ambiguous or otherwise unclear here?

Reply:
I think I may have been misinterpreting it as "If you're using this to serialize GUIDs then you need to register it. If you're using it for other reasons then you don't" or something of that sort, which I think stemmed from my confusion on the serializing objects vs serializing guids sections. Given the context in the section now I think it makes sense as is